src/components/academies/styles/Academy.scss

```diff
@@ -78,3 +78,17 @@
     }
   }
 }
+
+.academy-title {
+  font-size: 55px;
```

Comment:
I would recommend using pre-defined classes instead of hard coding values here.

Reply:
You can find classes reference at https://paragon-openedx.netlify.app/foundations/css-utilities/

Reply:
@saleem-latif I think we don't have any predefined classes that targets a font size of 55 px or closer to that...

Reply:
Yes, but there are classes to handle most of the definitions below. You should replace as many hardcoded definitions from here as possible.